src/main/java/com/footbalmanager/app/services/SeasonServiceImpl.java

```diff
@@ -0,0 +1,54 @@
+package com.footbalmanager.app.services;
+
+import com.footbalmanager.app.domain.Season;
+import com.footbalmanager.app.dto.season.PatchSeasonRequestDto;
+import com.footbalmanager.app.dto.season.PostSeasonRequestDto;
+import com.footbalmanager.app.repository.SeasonRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service("seasonService")
+@Transactional
+public class SeasonServiceImpl implements SeasonService {
```

Comment:
Tak się zastanawiam czy potrzebujemy tą warstwę serwisów które są tak naprawdę przelotką do repo? Wiem, że tak się czasami robi, ale w jednym z projektów zrezygnowaliśmy z tego i używaliśmy bezpośrednio repo, a serwisy tylko wtedy gdy faktycznie była tam jakaś logika biznesowa, której repo nie powinno znać/obsługiwać.

Reply:
Możemy z biegiem dodawania ficzerów uszczuplić ich ilość w taki sposób jak mówisz (bo o to chodzi - nadmiarowy w sumie kod na chwilę obecną). Zawsze wydawało mi się, że w przypadku samych repo w springu nie można przetestować kontrolerów jednostkowo, bo nie możesz ich zamockować w taki sposób jak robię to z serwisem obecnie i to główny argument czemu zawsze się z taką konwencją spotykałem. Widziałem też kod z repozytoriami i logiką w kontrolerach ale raczej uczono mnie, że to be :P

Reply:
Tak, logika w controllerach to "be" ale jeśli mamy cruda, to de facto można pominąć warstwę serwisów.